01/index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+¿Cuáles son los comandos más utilizados?
+mkdir - para crear una carpeta
+touch - para crear un archivo
+ls - para ver lo que hay dentro de una carpeta
+pwd - para mostrar la ruta 
+open - para abrir finder
+mv - para mover archivos
+git init - para iniciar un repositorio en git
+code .. - abre visual studio
+command + s - para guardar cambios
+git add . - para agregar los archivos que han tenido cambios
+git commit -m "___" - para agregar un commit
+git branch -v muestra el nombre de la rama y el último commit
+git push -u origin "nombre de la rama" - para enviar o subir cambios a github
```

Comment:
There are mixed examples of git and terminal and the exercise is just requesting terminal commands.